Library/Formula/libghthash.rb

```diff
@@ -0,0 +1,28 @@
+class Libghthash < Formula
+  homepage "http://www.bth.se/people/ska/sim_home/libghthash.html"
+  url "http://www.bth.se/people/ska/sim_home/filer/libghthash-0.6.2.tar.gz"
+  version "0.6.2"
```

Comment:
`brew audit` will probably complain that the version is correctly parsed from the URL so the `version` line is redundant and should be removed.